dev/src/index.js

```diff
@@ -591,15 +591,35 @@ class Firestore {
    * with an array of CollectionReferences.
    *
    * @example
+   * firestore.listCollections().then(collections => {
+   *   for (let collection of collections) {
+   *     console.log(`Found collection with id: ${collection.id}`);
+   *   }
+   * });
+   */
+  listCollections() {
+    let rootDocument = new DocumentReference(this, this._referencePath);
+    return rootDocument.listCollections();
+  }
+
+  /**
+   * Fetches the root collections that are associated with this Firestore
+   * database.
+   *
+   * @deprecated Use `listCollections()`.
+   *
+   * @returns {Promise.<Array.<CollectionReference>>} A Promise that resolves
+   * with an array of CollectionReferences.
+   *
+   * @example
```

Comment:
FWIW, I think it'd be fair to omit example code for deprecated methods.  I don't care strongly though.

Reply:
Done